clash-prelude/src/Clash/Magic.hs

```diff
@@ -30,6 +30,9 @@ module Clash.Magic
   -- ** Functions to control Clash's (de)duplication mechanisms
   , deDup
   , noDeDup
+
+  -- ** Utilities to differentiate between simulation and normalization
```

Comment:
Soft suggestion: normalization -> generating HDL. No need to confuse people new to the jargon with yet another.